src/ubuntu/16.04/cross/14.04/hooks/pre-build

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$(dirname ${BASH_SOURCE[0]})/../../../build-scripts/build-rootfs.sh ubuntu-14.04 trusty
```

Comment:
I've just created a PR for updating this script in the original 14.04 form (https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/145). Can you please copy the change here?